sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveSharedStateSuite.scala

```diff
@@ -88,4 +89,17 @@ class HiveSharedStateSuite extends SparkFunSuite {
     assert(!ss.conf.get(WAREHOUSE_PATH).contains(invalidPath),
       "session level conf should be passed to catalog")
   }
+
+  test("SPARK-34568: We should respect enableHiveSupport when initialize SparkSession") {
```

Comment:
It seems this test can pass in the current master. Have you checked that?

Reply:
> It seems this test can pass in the current master. Have you checked that?

Hmmm, make a mistake, we should check SessionState.., Updated